src/main/kotlin/io/easybreezy/project/model/issue/Priority.kt

```diff
@@ -43,16 +43,26 @@ class Priority private constructor() : Embeddable() {
             return priority
         }
 
-        fun highest(lastHighest: Int): Priority {
+        fun highest(lastHighest: Int?): Priority {
             val priority = Priority()
```

Comment:
Hmm, As I see every new issue with the highest priority will be on top on the board? What I should do when I need to change the position of these issues?

Reply:
Yes, by adding a comment "the highest priority" to the highest
one